habitLoggerApp.Natephor/Program.cs

```diff
@@ -0,0 +1,50 @@
+﻿using HabitLoggerApp;
+
+var userInterface = new UserInterface();
+var habitManager = new HabitManager();
+
+var appIsRunning = true;
+while (appIsRunning)
+{
+    userInterface.DisplayMenu(true);
+
+    while (appIsRunning)
+    {
+        var input = userInterface.GetUserInput("Enter your choice: ");
+        switch (input)
+        {
+            case "1":
+                var habits = habitManager.GetAllHabits();
+                userInterface.DisplayHabits(habits);
+                break;
+            case "2":
+                var body = userInterface.GetUserInput("Enter habit body: ");
+                var parsedDate = userInterface.GetUserDateInput("Enter habit date: ");
+                var quantity = userInterface.GetUserNumberInput("Enter quantity: ");
+
+                habitManager.CreateHabit(parsedDate, body, quantity);
+                break;
+            case "3":
+                var updatingId = userInterface.GetUserNumberInput("Enter the id to be updated: ");
+                var updatedBody = userInterface.GetUserInput("Enter habit body: ");
+                var updatedDate = userInterface.GetUserDateInput("Enter habit date: ");
+                var updatedQuantity = userInterface.GetUserNumberInput("Enter quantity: ");
+                habitManager.UpdateHabit(updatingId, updatedBody, updatedDate, updatedQuantity);
+                Console.WriteLine("Habit updated successfully!");
+                break;
+            case "4":
+                var deletionId = userInterface.GetUserNumberInput("Enter habit id for deletion: ");
```

Comment:
🟠 **No ID Validation**

💡 You have not validated that this ID exists. I know nothing has been deleted, but it still tells me that it successfully has 🙃.

![image](https://github.com/user-attachments/assets/66a5d74f-2de3-4a93-8ccd-3f8cc987e211)